dnstable_convert.c

```diff
@@ -100,7 +101,8 @@ static struct timespec		start_time;
 static uint64_t			count_messages;
 static uint64_t			count_entries_dns;
 static uint64_t			count_entries_dnssec;
-static uint64_t			count_entries_merged;
+/* Count merged entries correctly even when MTBL multithreading is enabled. */
+static atomic_uint_fast64_t	count_entries_merged;
```

Comment:
I think you need to use special functions to access this variable.
From nmsg library, we use:
./nmsg/io.c:	*sum_out = atomic_load_explicit(&io->io_count_nmsg_payload_out, memory_order_relaxed);
./nmsg/io.c:		uint64_t pl_out = atomic_load_explicit(&(*io)->io_count_nmsg_payload_out, memory_order_relaxed);
./nmsg/io.c:	atomic_fetch_add_explicit(&io->io_count_nmsg_payload_out, 1, memory_order_relaxed);